exotica_core/src/planning_problem.cpp

```diff
@@ -294,6 +294,11 @@ std::pair<std::vector<double>, std::vector<double>> PlanningProblem::GetCostEvol
     return ret;
 }
 
+int PlanningProblem::GetNumberOfIterations() const
+{
+    return static_cast<int>(std::get<0>(GetCostEvolution())).size();
```

Comment:
No, not like this. The static cast needs to close after size...

Reply:
Sorry, should have copy/paste..resolved